pkgs/jni/tool/generate_ide_files.dart

```diff
@@ -1,66 +0,0 @@
-// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
-
-import 'dart:io';
-
-import 'package:args/args.dart';
-
-const makefilesGenerator = 'make';
-const ninjaGenerator = 'ninja';
-
-const cmakeGeneratorNames = {
-  makefilesGenerator: 'Unix Makefiles',
-  ninjaGenerator: 'Ninja',
-};
-
-void runCommand(String exec, List<String> args, String workingDirectory) {
-  stderr.writeln('+ $exec ${args.join(" ")}');
-  final process =
-      Process.runSync(exec, args, workingDirectory: workingDirectory);
-  if (process.exitCode != 0) {
-    stdout.writeln(process.stdout);
-    stderr.writeln(process.stderr);
-    stderr.writeln('command failed with exit code ${process.exitCode}');
-    exit(1);
-  }
-}
-
-void main(List<String> arguments) {
-  final argParser = ArgParser()
-    ..addOption(
-      'generator',
-      abbr: 'G',
-      help: 'Generator to pass to CMake. (Either "ninja" or "make").',
-      allowed: [ninjaGenerator, makefilesGenerator],
-      defaultsTo: Platform.isWindows ? ninjaGenerator : makefilesGenerator,
-    )
-    ..addFlag('help', abbr: 'h', help: 'Show this usage information.');
-  final argResults = argParser.parse(arguments);
-  if (argResults.rest.isNotEmpty || argResults['help'] as bool) {
-    stderr.writeln('This script generates compile_commands.json for '
```

Comment:
We still have C sources, why are we deleting this?

Reply:
I never used this, I could keep it if you think it's useful. Do other packages that have C sources also have such a tool?

Reply:
This is what sets up the static analysis in vscode for clangd, please keep it.